app/docs/1.4.x/security/api-server-auth.md

```diff
@@ -68,9 +71,10 @@ kumactl config control-planes add \
   --auth-conf token=<GENERATED_TOKEN> \
   --ca-cert-file=/path/to/ca.crt # or --skip-verify if you want to skip CP verification
 ```
-
+{% endcapture %}
+{{ item | indent }}
```

Comment:
**Question**: Is this necessary because the code blocks don’t work when indented? Or what’s the reason for not _actually_ indenting this block correctly?

This would make for quite a frustrating editing experience to always be forced to capture and filter-indent list item content.

Reply:
oh my bad, we don't need to indent everything.
Just `tabs` and some code snippets, I'll update the PR with just the minimal changes

Reply:
fixed